modules/ROOT/pages/tutorials/upgrade.adoc

```diff
@@ -0,0 +1,174 @@
+* Use TEXT indexes if the property value type is `text` and the values can be larger than 8Kb.
+* Use TEXT indexes if the property value type is `text` and `CONTAINS` and `ENDS WITH` are used in queries for the property.
+
+After creating the new index, the old index should be dropped. 
+The following example shows how to create a new RANGE index and drop an existing `index_name` index:
+
+[source, Cypher, role="noplay"]
+----
+CREATE RANGE INDEX range_index_name FOR (n:Label) ON (n.prop1);
+DROP INDEX index_name;
+----
+
+The following example instead shows how to create a constraint backed by a RANGE index:
+
+[source, Cypher, role="noplay"]
+----
+CREATE CONSTRAINT constraint_with_provider FOR (n:Label) REQUIRE (n.prop1) IS UNIQUE OPTIONS {indexProvider: 'range-1.0'}
+----
+
+For more information about creating indexes, see link:https://neo4j.com/docs/cypher-manual/current/indexes-for-search-performance/#administration-indexes-examples[Cypher Manual -> Creating indexes].
+
+=== Cypher updates
+
+Neo4j latest version causes changes to the Cypher syntax and error handling.
+
+==== Cypher syntax
+
+All changes in the Cypher language syntax are detailed in link:https://neo4j.com/docs/cypher-manual/5/deprecations-additions-removals-compatibility[Cypher Manual -> Removals, deprecations, additions and extensions].
+Thoroughly review this section in the version you are moving to and make the necessary changes in your code.
+
+Here is a short list of the main changes introduced in Neo4j 5:
+
+[cols="1a,1a", options="header"]
+|===
+|*Deprecated feature*
+|*Details*
+
+|[source, Cypher, role="noplay"]
+----
+MATCH (n)-[r:REL]->(m) SET n=r
+----
+|Use the `properties()` function instead to get the map of properties of nodes/relationships that can then be used in a `SET` clause:
+
+[source, Cypher, role="noplay"]
+----
+MATCH (n)-[r:REL]->(m) SET n=properties(r)
+----
+
+|[source, Cypher, role="noplay"]
+----
+MATCH (a), (b), allShortestPaths((a)-[r]->(b)) RETURN b
+
+MATCH (a), (b), shortestPath((a)-[r]->(b)) RETURN b
+----
+|`shortestPath` and `allShortestPaths` without link:https://neo4j.com/docs/cypher-manual/5/syntax/patterns/#cypher-pattern-varlength[variable-length relationship] are deprecated. Instead, use a `MATCH` with a `LIMIT` of 1 or:
+[source, Cypher, role="noplay"]
+----
+MATCH (a), (b), shortestPath((a)-[r*1..1]->(b)) RETURN b
+----
+
+|[source, Cypher, role="noplay"]
+----
+CREATE DATABASE databaseName.withDot ...
+----
+|Creating a database with unescaped dots in the name has been deprecated, instead escape the database name:
+[source, Cypher, role="noplay"]
+----
+CREATE DATABASE `databaseName.withDot` ...
+----
+|===
+
+==== Error handling in Cypher
+
+Many semantic errors that Cypher finds are reported as `Neo.ClientError.Statement.SyntaxError` even though they are semantic and not syntax errors.
+In Neo4j latest version, the metadata returned by Cypher queries is improved.
```

Comment:
Im not sure what the correct term is but I dont think its "Info"

```suggestion
The following warnings in Neo4j 4.4 have since been changed into....```